ParisBNF/abb/BNFabb166.xml

```diff
@@ -0,0 +1,500 @@
+                                                                    <locus from="2ra" to="149vc"/>                                                                    
+                                                                    <ab type="pricking">Pricking is visible.</ab>
+                                                                    <ab type="ruling">Ruling is visible.</ab>
+                                                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>                                                                   
+                                                                </layout>
+                                                            </layoutDesc>                                              
+                                                    </objectDesc>
+                                                    
+                                                    <handDesc>
+                                                           <handNote xml:id="h2" script="Ethiopic"> 
+                                                                <locus from="5ra" to="15vc"/>                                
+                                                                <seg type="script">18th century</seg>                         
+                                                                <seg type="ink">Black; red.</seg>                     
+                                                                <desc>Careful, very regular; by a well-trained scribe.</desc>
+                                                            </handNote>
+                                                            
+                                                            
+                                                            <handNote xml:id="h3" script="Ethiopic">
+                                                                <locus from="16ra" to="149vc"/>
+                                                                <seg type="script">18th century</seg>                               
+                                                                <seg type="ink">Black; red.</seg>                     
+                                                                <desc>Less careful and less regular than <ref target="#h1">Hand 2</ref>/ref>; 
+                                                                    by a mediocre scribe, still similar to <ref target="#h1">Hand 2</ref>.
+                                                                    The involvement of more scribal hands cannot be excluded.</desc>
+                                                            </handNote>
+                                                    </handDesc>
+                                                    
+                                                    <additions>
```

Comment:
in your Oxygen editor, press the button
![image](https://github.com/BetaMasaheft/Manuscripts/assets/17987728/b9b2ae97-0e03-4164-8238-3dd79b299224)
to get a nicely nested XML